src/auralis/models/xttsv2/utils/checkpoint_converter.py

```diff
@@ -104,12 +106,13 @@ def convert_checkpoint(pytorch_checkpoint_path, output_dir, args):
         download_repo_files("AstraMindAI/xttsv2", os.path.join(output_dir, "core_xttsv2"))
         print(f"Saved XTTSv2 weights to {xtts_path}")
         print(f"XTTSv2 weights: {list(xtts_weights.keys())}")
-
 def main():
     parser = argparse.ArgumentParser(description='Convert PyTorch checkpoint to SafeTensors format')
     parser.add_argument('checkpoint_path', type=str, help='Path to PyTorch checkpoint file')
     parser.add_argument('--output_dir', type=str, default=os.getcwd(),
                         help='Output directory (defaults to current working directory)')
+    parser.add_argument('--tokenizer_size', type=int, default=None,
```

Comment:
Here instead of passing of passing as an argument you should take the vocab size form the embedding layer(check for different name types)